tests/base_case.py

```diff
@@ -70,4 +70,3 @@ def setUp(self):
         self.chatbot.train(data1)
         self.chatbot.train(data2)
         self.chatbot.train(data3)
-
```

Comment:
I don't know why this is happening because Atom shows a blank line at the end of the file, but I will see if I can fix this in a future commit